test/gtest/common/test_perf.cc

```diff
@@ -196,6 +196,9 @@ test_perf::test_result test_perf::run_multi_threaded(const test_spec &test, unsi
     strncpy(params.uct.tl_name , tl_name.c_str(),  sizeof(params.uct.tl_name));
     params.uct.data_layout = test.data_layout;
     params.uct.fc_window   = UCT_PERF_TEST_MAX_FC_WINDOW;
+    params.msg_size_lst    = NULL;
```

Comment:
I think it would be better to implement it as separate PR. I created the issue #1186 for this. 
It might be good idea to label it as "training"